src/main/java/com/mg105/utils/TutorialTexts.java

```diff
@@ -2,30 +2,36 @@
 
 import java.util.Arrays;
 import java.util.List;
-import java.util.concurrent.Phaser;
 
 public class TutorialTexts {
     public static final String moved = "moved";
     public static final String attacked = "attacked";
     public static final String usedItem = "usedItem";
 
-    public static final int tutorialTextX = 150;
-    public static final int tutorialTextY = 550;
+    public static final int tutorialTextX = 660;
+    public static final int tutorialTextY = 780;
 
-    public static final int TEXT_DURATION1 = 5;
-    public static final int TEXT_DURATION2 = 10;
+    public static final int helperPaneX = 500;
+    public static final int helperPaneY = 100;
 
-    public static final String textFont = "Verdana";
-    public static final int textSize = 18;
+
+    public static final int TEXT_DURATION1 = 4;
+
+    public static final String textFont1 = "Verdana";
+    public static final String textFont2 = "Courier";
```

Comment:
Beware that not everyone has these fonts. Maybe we should bundle our own? Maybe we should just allow the fallback font to show...